modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/paywallfeatures/FeaturesPaywallCards.kt

```diff
@@ -0,0 +1,225 @@
+package au.com.shiftyjelly.pocketcasts.account.onboarding.upgrade.paywallfeatures
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP50
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import au.com.shiftyjelly.pocketcasts.images.R as IR
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+import au.com.shiftyjelly.pocketcasts.ui.R as UR
+
+data class CardData(
+    val imageResId: Int,
+    val contentDescription: String,
+    val title: String,
+    val description: String,
+    val imageHeight: Dp,
+)
+
+@Composable
+fun FeaturePaywallCard(cardData: CardData) {
+    Card(
+        shape = RoundedCornerShape(8.dp),
+        modifier = Modifier
+            .height(394.dp)
+            .width(313.dp)
+            .background(Color.Transparent),
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(colorResource(UR.color.coolgrey_90)),
+        ) {
+            Image(
+                painter = painterResource(cardData.imageResId),
+                contentDescription = cardData.contentDescription,
+                modifier = Modifier
+                    .height(cardData.imageHeight)
+                    .padding(top = 20.dp, start = 2.dp, end = 2.dp)
```

Comment:
I am checking with our designs which top padding fits the best. When I add the same specs in figma it does not look great